README.md

```diff
@@ -565,3 +565,7 @@ harshithap2000-dai_deprivation"
 "Ckbhatia_flying_dino"
 
 ![](Ckbhatia_flying_dino.png)
+
+"rakshamaloo_dino"
+
+![](rakshamaloo_dino.png)
```

Comment:
Your drawing is really cool! However, you called the file `PicsArt_06-23-10.39.18.jpg`, so you have to change `![](rakshamaloo_dino.png)` to `![](PicsArt_06-23-10.39.18.jpg)`

Reply:
Okay